.github/workflows/check-migrations.yml

```diff
@@ -77,3 +77,11 @@ jobs:
           node-uri: ${{ matrix.runtime.uri }}
           checks: "pre-and-post"
           extra-args: ${{ env.EXTRA_ARGS }}
+
+  confirmMigrationsPassed:
+    runs-on: ubuntu-latest
+    name: All migrations passed
+    # If any new job gets added, be sure to add it to this list
```

Comment:
You should put some docs at the top of the file to make this more obvious!

Reply:
Good idea. I properly documented all the steps.

Let me know if there is anything else missing 🙂